include/orc/orc.hpp

```diff
@@ -18,8 +18,7 @@
 
 struct odrv_report {
     std::string_view _symbol;
-    die _a;
-    die _b;
+    const die* _die{nullptr};
```

Comment:
An ODRV report now only needs one `die` now - a pointer to the head node of the poisoned linked list.